runtime/doc/lsp.txt

```diff
@@ -1630,6 +1630,7 @@ Lua module: vim.lsp.completion                                *lsp-completion*
     Fields: ~
       • {autotrigger}?  (`boolean`) Whether to trigger completion
                         automatically. Default: false
+      • {abbr_format}?  (`function`)
```

Comment:
```suggestion
      • {abbr_format}?  (`function`) Decides the format of ...
```

Reply:
Mentioning the default format would also be helpful.